### ISSUE TITLE
0014: Frontier: assign work to workers

### ISSUE THREAD

**Issue by earthquakesan** [AKSW/Squirrel#3]
As a Frontier I want to be able to give a new task to a worker that is asking for work. In detail, this means:

get the element with the highest score from the queue
check whether the IP is already blocked by another worker. If it is blocked, ask the queue for the next element
mark the IP of the work package as blocked
give the work package to the worker
Note: this part might have to be sinchronized.
